src/hooks/sidebarTabs/modelLibrarySidebarTab.ts

```diff
@@ -18,7 +19,11 @@ export const useModelLibrarySidebarTab = (): SidebarTabExtension => {
     iconBadge: () => {
       if (isElectron()) {
         const electronDownloadStore = useElectronDownloadStore()
-        if (electronDownloadStore.downloads.length > 0) {
+        if (
+          electronDownloadStore.downloads.filter(
```

Comment:
Please consider add a new computed field `inProgressDownloads` in the electron download store.